docs/features/pouch_with_plugin.md

```diff
@@ -1,148 +1,159 @@
-# PouchContainer with plugin
+# Pouch-container with plugin
```

Comment:
Why to change the name of it? 
I think the logo of this project is PouchContainer indeed.

Reply:
Should pouchcontainer instead of pouch-container?